src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped work");
```

Comment:
Similarly, the message "Excavator stopped work" should be more informative. Indicate that it is the Excavator machine that has stopped working.